manifest-references-api.md

```diff
@@ -0,0 +1,113 @@
+# Manifest References API
+
+[OCI artifact-manifest](./artifact-manifest.md) provides the ability to reference artifacts to existing artifacts. Reference artifacts include Notary v2 signatures, SBoMs Nydus expansion images and many other types. Artifacts that reference others SHOULD not be tagged, as they are considered enhancements to the artifacts they reference. To discover referenced artifacts a manifest references API is provided. An artifact client, such as a Notary v2 client would parse the returned manifests, determining which manifest type they will pull and process.
+
+The `references` API returns all artifacts that are linked a given manifest digest, with optional filtering. Referenced artifact requests are scoped to a repository, ensuring access rights for the repository can be used as authorization for the referenced artifacts.
+
+Artifact references are defined in the [oci.artifact.manifest spec][oci.artifact.manifest-spec] through the [`[manifests]`][oci.artifact.manifest-spec-manifests] collection.
+
+## Request All Artifact References
+
+The references api is defined as an extension to the [distribution-spec][oci-distribution-spec] using the `/v2/_ext/oci-artifacts/v1/` namespace. This spec defines the behavior of the `v1` version. Clients MUST account for version checking as future major versions MAY NOT be compatible. Future Minor versions MUST be additive.
+
+The `/references` API MUST provide for paging. The default page size SHOULD be set to 10.
+
+```rest
+GET /v2/_ext/oci-artifacts/v1/{repository}/manifests/{digest}/references?n=10
+```
+
+**expanded example:**
+
+```rest
+GET /v2/_ext/oci-artifacts/v1/net-monitor/manifests/sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b/references?n=10
+```
+
+The `/references` API MAY provide for filtering of artifactTypes. Artifact clients MUST account for [distribution-spec][oci-distribution-spec] implementations that MAY NOT support filtering. Artifact clients MUST revert to client side filtering to determine which artifactTypes they will process.
```

Comment:
Calling this API `/referrers` would better capture the relationship between the returned data (a collection of artifacts that refer to this resource) and the manifest.
`references` could easily be confused to mean "the artifacts this manifest references...", when we actually mean "the artifacts that reference this manifest".

Reply:
changed to `referrers`